mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -342,7 +342,12 @@ class GBTClassificationModel private[ml](
     }
   }
 
-  /** Number of trees in ensemble */
+  /**
+   * Number of trees in ensemble
+   *
+   * @deprecated  Use [[getNumTrees]] instead.  This method will be removed in 3.1.0
+   */
+  @deprecated("Use getNumTrees instead. This method will be removed in 3.1.0.", "3.0.0")
```

Comment:
I'm wondering if we can deprecate this in 2.4.5 and remove in 3.0.0.

Reply:
so deprecate this in v2.4.5-rc1, right?

Reply:
It's `2.4.5` because RC is not an official release.
RC1 vote fails and we are preparing 2.4.5 RC2.

Reply:
Got it. Thanks!